### DISCUSSION TITLE
[cnx] Roguelike地图游走 / 8行拼字

### DISCUSSION THREAD

**Discussion by qiufuyu123** [Physics365/991CN-X-CW-Decompilation#45] [Ideas]
https://github.com/Physics365/991CN-X-CW-Decompilation/assets/60765480/babbadf5-140d-4c21-9394-f41bc50a7b5f

**Comment by qiufuyu123** [comment]
[程序在这](https://github.com/Physics365/991CN-X-CW-Decompilation/blob/main/ROP/ROPCodes/Examples/game.asm)

**Reply by qiufuyu123** [reply]
使用最小字体(8pixels 表格字体）绘制地图
可以显示32*8=256字符

**Comment by fish4terrisa-MSDSM** [comment]
感觉rogue或者hack指日可待的说（是不是可以随机生成地图？

**Comment by qiufuyu123** [comment]
下一步是想想怎么优化这一块
![image](https://github.com/Physics365/991CN-X-CW-Decompilation/assets/60765480/d56084b9-38c4-4433-a30b-6971cbaa7381)
目前思路是硬编码8次smallprint
可能用更好的gadget可以实现
